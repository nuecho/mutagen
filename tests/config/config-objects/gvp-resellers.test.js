cfgObjectTests("gvpResellers", "config-objects/gvp-resellers-config.json");