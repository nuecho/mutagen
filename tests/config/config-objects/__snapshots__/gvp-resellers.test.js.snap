// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[mutagen gvpResellers config import-export] should create the gvpResellers: the gvpResellers have been created 1`] = `
Object {
  "stderr": "",
  "stdout": "+ gvpreseller [Environment/cfgObjectTest]
Completed. 1 object created. 0 object updated. 0 object skipped.
",
}
`;

exports[`[mutagen gvpResellers config import-export] should skip the same gvpResellers: the gvpResellers have been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "= gvpreseller [Environment/cfgObjectTest]
Completed. 0 object created. 0 object updated. 1 object skipped.
",
}
`;

exports[`[mutagen gvpResellers config import-export] should update the modified gvpResellers: the modified gvpResellers have been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "~ gvpreseller [Environment/cfgObjectTest]
Completed. 0 object created. 1 object updated. 0 object skipped.
",
}
`;
